loader/test/albertsons.test.js

```diff
@@ -206,6 +212,83 @@ describe("Albertsons", () => {
     expect(result[0]).toHaveProperty("availability.products", ["pfizer"]);
   });
 
+  it("handles separate adult and pediatric entries for the same location", async () => {
+    nock(API_URL_BASE)
+      .get(API_URL_PATH)
+      .query(true)
+      .reply(
+        200,
+        [
+          {
+            id: "1635993536219",
+            region: "Eastern_-_6",
+            address:
+              "Pfizer Child - Safeway 0005 - 11120 South Lakes Drive, Reston, VA, 20191",
+            lat: "38.939784",
+            long: "-77.332298",
+            coach_url: "https://kordinator.mhealthcoach.net/vcl/1635993536219",
+            availability: "yes",
+            drugName: ["PfizerChild"],
+          },
+          {
+            id: "1600100807144",
+            region: "Virginia",
+            address:
+              "Safeway 0005 - 11120 South Lakes Drive, Reston, VA, 20191",
+            lat: "38.939784",
+            long: "-77.332298",
+            coach_url: "https://kordinator.mhealthcoach.net/vcl/1600100807144",
+            availability: "yes",
+            drugName: ["Pfizer", "Moderna", "JnJ"],
+          },
+        ],
+        { "Last-Modified": "Thu, 28 Oct 2021 07:06:13 GMT" }
+      );
+
+    const result = await checkAvailability(() => {}, { states: "VA" });
+    expect(result).toContainItemsMatchingSchema(locationSchema);
+    expect(result).toEqual([
+      {
+        name: "Safeway 0005",
+        external_ids: [
+          ["albertsons", "1635993536219"],
+          ["albertsons_safeway", "1635993536219"],
+          ["safeway", "0005"],
+          ["albertsons_store_number", "safeway:0005"],
```

Comment:
It *seemed* useful to surface this `albertsons_store_number` field as an external ID, but it’s actually only used for identifying two locations that need merging. This could also be in `meta` (which would simplify some code), or hidden from the output entirely.